src/components/Table/index.tsx

```diff
@@ -18,14 +18,16 @@ const Table: ((props: BoxProps) => JSX.Element) & {
   BodyCell: typeof BodyCell
 } = props => <StyledTable {...props} />
 
-export const Head = styled(Box.withComponent('thead'))`
+const StyledHead = styled(Box.withComponent('thead'))`
   border: 0;
   border-bottom-width: 1px;
   border-color: ${({ theme }) => theme.colors.neutral.borderWeak};
   border-style: solid;
 `
 
-export const Row = styled(Box.withComponent('tr'), {
+export const Head = (props: BoxProps) => <StyledHead {...props} />
```

Comment:
What happens if we do 
?
```ts
export const Head = StyledHead 
```

Reply:
@chambo-e Still same issue

Reply:
Can we raise an issue on emotion or docgen in the meantime ? This is opposite of everything we are trying to do and external tools should not impact our way of doing things :)